packages/graph/hash_graph/lib/graph/src/shared/identifier/time/projection.rs

```diff
@@ -218,6 +218,9 @@ impl ToSchema for UnresolvedTimeProjection {
     }
 }
 
+/// The pinned axis of a [`TimeProjection`].
```

Comment:
If we have the term `pinned axis` for `kernel`, why do we not call it `PinnedAxis` instead? Not sure if this was brought up before :)